.travis.yml

```diff
@@ -39,8 +39,9 @@ before_script:
 script:
   - bin/check-readme
   - bin/code-analysis
-  - bin/nosetests
   - bin/test
+  - bin/createcoverage
```

Comment:
I think createcoverage should run in the after_success sestion. This is not a test.